zingolib/src/wallet/notes/query.rs

```diff
@@ -38,13 +38,13 @@ impl OutputSpendStatusQuery {
 pub struct OutputPoolQuery {
     /// will the query include transparent notes? (coins)
     #[getset(get = "pub")]
-    transparent: bool,
+    pub transparent: bool,
```

Comment:
i dont understand why the pub getset isnt good enough and how making these pub relates to this PR? i probably have missed something

Reply:
its so that we can use naive struct instantiation

Reply:
oh, i guess i didnt end up using it in this branch ...